### ISSUE TITLE
Sync downloaded video to my PMS.

### ISSUE THREAD

**Issue by ppgub** [NineWorlds/serenity-android#74]
Is there any synchronization between videos downloaded to device and PMS?

The desired behavior is this:
- when I download a video to device with Serenity and watch it offline, next time I connect to my LAN, I would like the file to be marked as watched on PMS
- when I download a video to device with Serenity and delete the local video from my device, next time I connect to my LAN, I would like a message to ask me to confirm to remove the video from PMS as well
- when I download a video to device with Serenity and watch the local video on my device to 03:54, next time I connect to my LAN, I would like the "Last Played Position" to be updated to 03:54 for the corresponding video in PMS.

In a nutshell can offline viewing of videos downloaded to device be handled entirely by Serenity and synced with PMS?

**Comment by kingargyle** [comment]
Out of scope for the version 2.0.0.  Only dedicated to playback from the server.
